css_stylesheet.css

```diff
@@ -35,7 +32,7 @@ header {
 	color: #343A40;
  }
  
-.contactAnchorsStyling:hover {
+.contactAnchorsStyling:hover, .contactAnchorsStyling:active {
```

Comment:
I don't know whether or not Ash will pick on you for this, but selectors of this sort might be more readable split across multiple lines - consider putting a line break after the comma

Reply:
Fixed, thanks.